Tests/Functional/ErrorHandling/main.swift

```diff
@@ -104,6 +108,20 @@ class ErrorHandling: XCTestCase {
         XCTAssertEqual(try functionThatShouldReturnButThrows(), 1)
     }
 }
+
+// CHECK: Test Case 'ErrorHandling.test_shouldNotThrowErrorDefiningSuccess' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ErrorHandling.test_shouldNotThrowErrorDefiningSuccess' passed \(\d+\.\d+ seconds\)
+func test_shouldNotThrowErrorDefiningSuccess() {
+    XCTAssertNoThrow(try functionThatDoesNotThrowError())
+}
+
+// CHECK: Test Case 'ErrorHandling.test_shouldThrowErrorDefiningError' started at \d+:\d+:\d+\.\d+
+// CHECK: .*/ErrorHandling/main.swift:[[@LINE+3]]: error: ErrorHandling.test_shouldThrowErrorDefiningFailure : XCTAssertNoThrow failed: threw error -
```

Comment:
One thing to keep in mind while you're working on the contents of this CHECK line is that the string is interpreted as a regex, so you may need to escape some characters to get the behavior you're looking for.